ci-operator/step-registry/interop/mtr/README.md

```diff
@@ -0,0 +1,97 @@
+# interop-mtr-chain<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+  - [MTR Chain Steps](#mtr-chain-steps)
+- [Requirements](#requirements)
+  - [Variables](#variables)
+  - [Infrastructure](#infrastructure)
+  - [Credentials](#credentials)
+
+## Purpose
+
+To prepare an environment and execute interop testing for the MTR (previously MTA) operator on the most recent version of OpenShift. The tests are maintained in the [windup/windup_integration_test](https://github.com/windup/windup_integration_test.git) repository by MTR product QE.
+
+This chain is to be used by the [windup-windup_integration_test-main](../../../config/windup/windup_integration_test/README.md) configuration
+
+## Process
+
+The process for this scenario attempts to follow a general flow of orchestrate, execute, report.
+
+- **Orchestrate**: Install any necessary operators and additional infrastructure for testing.
+- **Execute**: Execute the tests from the test repository.
+- **Report**: Publish XUnit results to Report Portal or any other platform needed.
+
+### MTR Chain Steps
+
+Below is a list of steps and sub-steps executed within this chain. Each step is linked to the step's README file, if you'd like to learn more about a step, please follow the links to those pages. Retrieve 
```

Comment:
I think that once we merge this code we will be able to replace this section with a link to the [auto-generated docs](https://steps.ci.openshift.org/) for this chain.

Reply:
Agreed, I'll keep this open and replace it when we are closer to merging.